### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by markedwardmurray**
Could be better overall, but I gotta turn it in cuz of time.

Still using some of Zach's screenshots and paired example code. Let me know if this doesn't work with the new flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,87 @@
...
+![](https://curriculum-content.s3.amazonaws.com/reading-ios-debugging/screenWithDebugConsole.png)
 
-Why `NSLog` may not always be the right choice:
+A pair of configuration buttons in the lower right of the Debug area governs these two subviews. You can show either the Variable Viewer alone, the Console Output Viewer alone, or both together. The bar separating them can be adjusted by dragging it left or right.
 
-1) In production code, you do not want any `NSLog` statements. Therefore some developers may choose to go with other debugging methods to avoid having to do code cleanup later.
+## Your Friendly Neighborhood `NSLog()`
 
-2) `NSLog` only gives you certain limited information, prior to runtime. If you are looking to further explore a variable at runtime beyond the initial `NSLog` code you wrote prior to compiling, you will have to use other methods.
+![][spiderman]
```
Comment (jmburges) [inline README.md:52]: Does this work?

Comment (jmburges) [inline README.md:52]: yes it does!

**Comment by jmburges** [comment]
I would add in the all exceptions breakpoint section somethiing on _when_ to use them. In general my rule is when you get that crash with the stack trace (screenshot!) you should enable the all exceptions breakpoint to see the line it happened on

**Comment by jmburges** [comment]
fix that and :shipit:

**Comment by markedwardmurray** [comment]
closes #4

**Pull request merged**